### ISSUE TITLE
[Android] Do not use intensive CPU

### ISSUE THREAD

**Issue by raphael-beck** [green-code-initiative/creedengo-challenge#28]
Using the full performance CPU will consume a lot of battery (POWER_HINT_SUSTAINED_PERFORMANCE cf https://source.android.com/docs/core/power/performance?hl=fr). 
In some cases using the low co-CPU will render the same service in an user acceptable time https://ieeexplore.ieee.org/stamp/stamp.jsp?tp=&arnumber=8808911

Team 23cm

**Comment by olegoaer** [comment]
Indeed. Detecting if `isSustainedPerformanceModeSupported()` return `True `seems enough to raise an issue.

**Comment by olegoaer** [comment]
Maybe it is preferable to check `Window.setSustainedPerformanceMode(true)`
